alg_convex_hull.go

```diff
@@ -0,0 +1,119 @@
+package simplefeatures
+
+import (
+	"sort"
+)
+
+const (
+	// Right indicates the orientation is right turn which is anticlockwise
+	Right = iota
```

Comment:
Could you make `Right`, `Collinear`, and `Left` unexported? They wouldn't be needed by library users.